### ISSUE TITLE
Respond with weather at [location]

### ISSUE THREAD

**Issue by brendanebers** [brendanebers/wheresbrendan#3]

**Comment by organicveggie** [comment]
What is the desired interface? You send an SMS to a number and it responds with the weather?

What formats for *location* do you want to support? Ideas include:

* GPS coordinates
* Airport code
* Text string containing things like "City, State, Country"

**Comment by brendanebers** [comment]
Yes, an SMS to a number, and it responds with weather information. The
location needs to support text, such as "Mexico city" or "San Fransisco". I
haven't looked into it yet, but twilio supports some form of cookies. If
there are multiple destinations like match the location string, it would be
nice to send a clarification text

1 San Fransisco, CA, US 4534 km NNW
2 San Francisco, Jal, MX 137 km NE

So I could respond "2" to get the appropriate results
